src/java.base/share/classes/java/util/Objects.java

```diff
@@ -189,6 +190,19 @@ public static String toIdentityString(Object o) {
         return o.getClass().getName() + "@" + Integer.toHexString(System.identityHashCode(o));
     }
 
+   /**
+    * {@return {@code true} if the object is an identity object, otherwise {@code false}}
```

Comment:
nit: s/the object/the specified object/

Reply:
The pattern in all of Objects.java is to refer to "the object".

Reply:
WIll fix to match other casesin Objects.java.